ulib/axstarry/src/syscall_fs/ctype/eventfd.rs

```diff
@@ -1,29 +1,48 @@
+use alloc::sync::Arc;
 use axerrno::AxResult;
 use axfs::api::{FileIO, FileIOType};
+use axsync::Mutex;
 
 pub struct EventFd {
-    value: u32,
+    value: Arc<Mutex<u64>>,
```

Comment:
Mutex may not be needed here as the whole EventFd object is already accessed exclusively already. Make it work firstly.

Reply:
The value may need the mutex lock, cause the eventfd will be read and written as a file descriptor by threads in the form of an Arc pointer. So the write function should be immutable

Reply:
Maybe it can add an attribute to indicate the flags of the fd. But if the FastDDS does't use the flag of the eventfd, tyou can choose whether to add it or not.(Sorry I don't analyze the syscall condition of FastDDS, so this is a suggestion.)

Reply:
> add an attribute to indicate the flags of the fd

Thanks. What do you mean by adding an attribute? @Azure-stars 

> But if the FastDDS does't use the flag of the eventfd

I am also not sure if FastDDS uses it or not, but I see the flag parameter has value. Will have deeper look into the flag handling. Thanks.